### ISSUE TITLE
Not an issue, just a request

### ISSUE THREAD

**Issue by moose4lord** [elliotforbes/ng-diff-match-patch#20]
Hello Elliot, I'm using your ng-diff-match-patch module to look at large JSON files that are mostly the same. The files are so large that I end up doing a lot of scrolling, so it would be nice to suppress lines that are the same in the HTML output and only show lines that are different. Sorry, I have no idea how this would be implemented, so can't help with that. Just thought I'd mention it would be a nice feature.

Thanks for sharing your work.

**Comment by elliotforbes** [comment]
I could take a look at implementing a compressed-line-diff directive which will achieve similar functionality. I can definitely see the benefit in implementing something like this, I'll try to pick this up at the weekend when I get some free time!

**Comment by moose4lord** [comment]
Thanks for looking into this, Elliot. I did download the project and noticed a lineComparator component that implements something similar to what I'm talking about. Although it's implemented as a component, not a directive, so seems out of place with the other parts of the project.
I did try to import the project to see if I could tweak the createHtml() function to do what I wanted,  but I've been using Angular 4 and the Angular CLI and this project seems quite different from what I'm used to. I couldn't get it to build in my Windows Visual Studio env. :(
